recipes/rockpool/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rockpool" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3ded42eba5e25abfda55f839ffe983010f423806dbfca961cba2321c00dc6c8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy
+    - scipy
+    - numba
+    - python >=3.6
+
+test:
+  requires:
```

Comment:
These shouldn't be required for the import test

Reply:
Removed